### DISCUSSION TITLE
Which operator version supports kafka-2.7.2？

### DISCUSSION THREAD

**Discussion by zhangyanfa** [strimzi/strimzi-kafka-operator#6640] [Q&A]
Which operator version supports kafka-2.7.2？

**Comment by scholzj** [comment]
I don't think we ever supported 2.7.2. The supported Kafka versions are listed here: https://strimzi.io/downloads/

**Reply by zhangyanfa** [reply]
Will 2.7.2 be supported in the future?

**Reply by scholzj** [reply]
I'm afraid it won't. Our policy given the amount of changes happening in Kafka is right now to support two latest minor versions. So right now 3.0.x and 3.1.x. The next release will add 3.0.1 and possibly 3.1.1 if it is already released.
